Install/RevitIFCSetupWix/Product.wxs

```diff
@@ -2,7 +2,7 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"

```

Comment:
Please back out any changes to configuration files, since they are irrelevant to the change.

Reply:
revert install project